uwblueprint/humane-society
# Create Users Table Skeleton

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Users-Table-Skeleton-4405603833e34267b0e1241ed5de6ef4?pvs=4)



## Implementation description
* This task just involved building the frontend table on the user management page, that displays all the users and user information. Does one call to the GET /user endpoint to get all the users upon page load. Designed the frontend using ChakraUI components, just basic frontend.

![image](https://github.com/user-attachments/assets/719dae17-d78d-4957-aa05-3dc1acef3c2e)



## Steps to test
1. Assume a role of animal behaviourist or administrator
2. Access the route: /admin/users
3. Test if you can get all the users and also press the refresh button to refresh users
4. Update the database (add users, delete users, edit user), should see if this change 
is reflected on the table


## What should reviewers focus on?
* Just ensuring that you can access and get the users in the table.


## Checklist
